client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -79,11 +79,11 @@ const CancelPurchaseButton = React.createClass( {
 				<p>
 					{ this.translate(
 						'Are you sure you want to cancel and remove %(purchaseName)s from {{em}}%(domain)s{{/em}}? ' +
-						'All plan features and custom changes will be removed from your site and you will be refunded %(priceText)s.', {
+						'All plan features and custom changes will be removed from your site and you will be refunded %(refundText)s.', {
```

Comment:
If we keep `priceText` in the string translators won't have to update it in all the languages we support. Also I think `priceText` better describes what is going to be outputted, i.e. a price with a currency.